libs/platform/table/components/table-view-settings-dialog/settings-dialog/settings-dialog.component.scss

```diff
@@ -0,0 +1,5 @@
+:host {
+    .fit-fd-segmented-button {
```

Comment:
not really a big deal, but please use a more proper naming. Something like `fd-segmented-button--fit-height` or similar.

Reply:
In such cases we can also apply inline CSS

Reply:
okay, i will remove scss file and set style inline. thanks.